src/Altinn.App.Core/Models/Layout/Components/PageComponent.cs

```diff
@@ -25,4 +23,12 @@ public class PageComponent : GroupComponent
     /// Helper dictionary to find components without traversing childern.
     /// </summary>
     public Dictionary<string, BaseComponent> ComponentLookup { get; }
+
+    /// <summary>
+    /// AddChild is not needed for PageComponent
+    /// </summary>
+    public override void AddChild(BaseComponent child)
```

Comment:
Not changed in this PR, but is it correct for the PageComponent to inherit from the GroupComponent in the first place? Or should it inherit directly from BaseComponent?

Reply:
I suppose it makes the logic simpler when you can just check if it `is GroupComponent` whenever you need to access children

Reply:
Do we want a accidental call to AddChild method on a PageComponent to result in a exception or should the method just not do anything?